packages/engine/src/avatar/components/AnimationComponent.ts

```diff
@@ -36,3 +42,34 @@ export const AnimationComponent = defineComponent({
     animations: S.Array(S.Type<AnimationClip>())
   })
 })
+
+export const useLoadAnimationFromBatchGLTF = (urls: string[], keepEntities = false) => {
+  const animations = urls.map((url) => useLoadAnimationFromGLTF(url, keepEntities))
```

Comment:
can we guarantee that `urls` does not change length? if not, we need to use a subreactor to handle this

Reply:
atm there is no use case I can think of for changing its length so I didn't bother with a subreactor. Currently its used for loading hard coded arrays of urls pointing to animation assets. not opposed to using a subreactor though as it might be useful in the future.